grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java

```diff
@@ -204,6 +204,114 @@
     CONCEPT_NOT_THING("Attempted concept conversion from concept [%s] that is not a thing."),
 
     //--------------------------------------------- Analytics Errors -----------------------------------------------
+    INVALID_COMPUTE_METHOD("Invalid compute method. " +
+            "The available compute methods are: count, min, max, median, mean, std, sum, path, paths, centrality, and cluster. " +
+            "A valid compute method contains 'compute <method> <conditions>', such as: 'compute min of age, in person;"),
+
+    // Graql compute count errors ------------------
+
+    INVALID_COMPUTE_COUNT_CONDITION("Invalid 'compute count' condition. " +
+            "Only 'in <types>' is accepted, such as: 'compute count in [person, movie]];'."),
```

Comment:
small typo: double closing bracket.

Reply:
I will fix this in the next PR, @lolski. So it doesn't have to trigger another test.

Reply:
sure

Reply:
okay i fixed it in the end, cause there were more improvements I wanted to add to the Graql.g4 grammar file. @lolski